### ISSUE TITLE
How to communicate between components

### ISSUE THREAD

**Issue by PierBover** [akxcv/vuera#37]
I'm using React in Vue.

Besides sending data to React with props and receiving events from React to Vue (eg:`@myEvent="someHandler"`), I'd like to be able to send events or call methods from the parent Vue component to the child React component.

In Vue this would be done with `this.$refs.component.method()`. I understand this is not an ideal approach from an architectural standpoint, but for certain use cases this is a valid approach.

So, how would this be accomplished with Vuera?

**Comment by TrumpClone** [comment]
luks gret! DU EET!

**Comment by PierBover** [comment]
> The thing to do in your case is to introduce a custom event bus or an event emitter, which your React component will subscribe to.

Yes, this is what I ended up doing. It's very bureaucratic though.

Generally in a Vue project I'd use Vuex, or very exceptionally refs.

Have you considered integrating Vuex with Reacts components instantiated through Vuera?

**Comment by akxcv** [comment]
Yes, integrating Vuex and Redux is something I thought about, but it's not clear what the API should look like. Especially since Vuex is so different from what React (or React with Redux) has to offer. If you have any ideas, please let me know.
